src/components/ClusterManager.js

```diff
@@ -533,10 +522,10 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     const totalCost = _.sum(_.map(({ machineConfig, status }) => {
       return (status === 'Stopped' ? machineStorageCost : machineConfigCost)(machineConfig)
     }, clusters))
-    const isDisabled = !canCompute
     const activeClusters = this.getActiveClustersOldestFirst()
     const creating = _.some({ status: 'Creating' }, activeClusters)
     const multiple = !creating && activeClusters.length > 1
+    const isDisabled = !canCompute || creating || multiple
```

Comment:
I think we usually have some sort of tooltip explaining why something is disabled if it is. Might be useful here too.